pkg/hostagent/hostagent.go

```diff
@@ -128,7 +128,8 @@ func New(instName string, stdout io.Writer, sigintCh chan os.Signal, opts ...Opt
 		}
 		vSockPort = port
 	} else if *y.VMType == limayaml.QEMU {
-		virtioPort = filenames.VirtioPort
+		// virtserialport doesn't seem to work reliably: https://github.com/lima-vm/lima/issues/2064
+		virtioPort = "" // filenames.VirtioPort
```

Comment:
Re-assigning the empty string here to satisfy the linter; couldn't get it to shut up blathering about empty branches and empty blocks if I left everything commented out...